### PR TITLE
<SectionHelper /> Component - Basics

### PR DESCRIPTION
**Pull request by liorzisman**
Implementing the <SectionHelper /> Component basics as requested.

### CODE REVIEW

File: src/SectionHelper/CloseButton/CloseButton.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import Button from '../../Button';
+import Close from '../../Icons/dist/components/Close';
+
+const CloseButton = () => (
+  <Button height="medium" theme="close-transparent" onClick={e => e.preventDefault()}>
+    <Close size="6px"/>
+  </Button>
+);
+
+CloseButton.displayName = 'Notification.CloseButton';
```
Comment (shlomitc) [inline src/SectionHelper/CloseButton/CloseButton.js]: SectionHelper.CloseButton

File: src/SectionHelper/SectionHelper.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  border-radius: 6px;
+  background-color: $Y50;
+  border: 1px solid $Y20;
+}
+
+.content {
+  padding: 30px 18px;
+}
+
+.closeButton {
```
Comment (shlomitc) [inline src/SectionHelper/SectionHelper.scss]: no close button yet

File: src/SectionHelper/SectionHelper.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  position: absolute;
+  top: 50%;
+  bottom: 50%;
+  right: 12px;
+  display: flex;
+  align-items: center;
+}
+
+/*Position*/
```
Comment (shlomitc) [inline src/SectionHelper/SectionHelper.scss]: all this section is redundant

File: src/SectionHelper/SectionHelper.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    );
+  }
+}
+
+SectionHelper.propTypes = {
+  theme: PropTypes.oneOf(Object.keys(HELPER_THEME))
+};
+
+SectionHelper.defaultProps = {
+  theme: HELPER_THEME.default
```
Comment (shlomitc) [inline src/SectionHelper/SectionHelper.js]: default is not defined there

File: src/SectionHelper/SectionHelper.spec.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import sectionHelperDriverFactory from './SectionHelper.driver';
+import {createDriverFactory} from '../test-common';
+
+import SectionHelper from './SectionHelper';
+
+const renderSectionHelperWithProps = (properties = {}) => (
+  <SectionHelper theme={properties.theme} data-hook="section-helper-wrapper">
```
Comment (shlomitc) [inline src/SectionHelper/SectionHelper.spec.js]: `dataHook`

File: src/SectionHelper/SectionHelper.spec.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    const driver = createDriver(renderSectionHelperWithProps());
+    expect(driver.exists()).toBeTruthy();
+  });
+
+  it('renders with provided content', () => {
+    const driver = createDriver(renderSectionHelperWithProps({children: 'Muffins are the best!'}));
+    expect(driver.textContent()).toEqual('Muffins are the best!');
+  });
+
+  describe('Themes', () => {
```
Comment (shlomitc) [inline src/SectionHelper/SectionHelper.spec.js]: default testing is missing

File: src/SectionHelper/SectionHelper.driver.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+const sectionHelperDriverFactory = ({element}) => {
+  const classExists = className => element.classList.contains(className);
+
+  return {
+    exists: () => !!element,
+    textContent: () => element.textContent,
+    hasTheme: theme => classExists(`${theme}`),
```
Comment (shlomitc) [inline src/SectionHelper/SectionHelper.driver.js]: redundant

File: src/SectionHelper/SectionHelper.driver.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    isStandard: () => classExists('standard'),
+    isDanger: () => classExists('danger'),
+    isSuccess: () => classExists('success'),
+    isDefault: () => classExists('default'),
```
Comment (shlomitc) [inline src/SectionHelper/SectionHelper.driver.js]: should verify it's danger

File: .storybook/stories.js [hunk 9]
```diff
@@ -114,3 +114,6 @@ import '../stories/PageHeader';
 
 // 11. Pickers and Selectors
 import '../stories/ColorPicker'; // 11.5 Color Picker
+
+// 12. SectionHelper
+import '../stories/SectionHelper';
```
Comment (shlomitc) [inline .storybook/stories.js]: EOL

File: src/SectionHelper/CloseButton/CloseButton.js (hunk 1 above)
Comment (liorzisman) [reply src/SectionHelper/CloseButton/CloseButton.js]: removed totally

File: src/SectionHelper/SectionHelper.driver.js (hunk 7 above)
Comment (liorzisman) [reply src/SectionHelper/SectionHelper.driver.js]: fixed

File: src/SectionHelper/SectionHelper.driver.js (hunk 8 above)
Comment (liorzisman) [reply src/SectionHelper/SectionHelper.driver.js]: better than `destructive` :) perhaps `error`?

File: src/SectionHelper/SectionHelper.js (hunk 4 above)
Comment (liorzisman) [reply src/SectionHelper/SectionHelper.js]: adjusted

File: src/SectionHelper/SectionHelper.scss (hunk 2 above)
Comment (liorzisman) [reply src/SectionHelper/SectionHelper.scss]: removed

File: src/SectionHelper/SectionHelper.scss (hunk 3 above)
Comment (liorzisman) [reply src/SectionHelper/SectionHelper.scss]: right

File: src/SectionHelper/SectionHelper.spec.js (hunk 6 above)
Comment (liorzisman) [reply src/SectionHelper/SectionHelper.spec.js]: added

File: .storybook/stories.js (hunk 9 above)
Comment (liorzisman) [reply .storybook/stories.js]: solved

**Pull request merged**